gravitee-apim-console-webui/src/management/api/creation/v4/models/ApiCreationStepperService.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Injectable, InjectionToken, Type } from '@angular/core';
+import { Subject } from 'rxjs';
+import { cloneDeep } from 'lodash';
+
+import { ApiCreationPayload } from './ApiCreationPayload';
+
+export const API_CREATION_PAYLOAD = new InjectionToken<ApiCreationPayload>('API_CREATION_PAYLOAD');
+
+export interface ApiCreationStep {
+  number: number;
+  component: Type<unknown>;
+  payload: ApiCreationPayload;
+  type: 'primary' | 'secondary';
```

Comment:
What is the purpose of the type `secondary`?

Reply:
it's for the sub step on Entrypoints for exemple